app/constants/constants.js

```diff
@@ -36,6 +37,7 @@ module.exports = {
   StatWrapFiles: {
     BASE_FOLDER: '.statwrap',
     LOG: '.statwrap.log',
+    CHECKLIST: '.statwrap.checklist',
```

Comment:
definitely ! it was never a good feeling for me to write some made up file extension, but i did, because my system doesn't recognize .log as well so i thought why not.. lol